src/main/webapp/index.html

```diff
@@ -671,7 +671,7 @@ <h4 class="title">Apache & OpenSource
     <div class="bottomcontant">
         <div class="container-fluid">
             <div class="col-md-8 col-sm-8">
-                <p class="copyright-txt">Copyright © 2017-2018 Apache CarbonData. All rights reserved &nbsp;&nbsp;|&nbsp;&nbsp;
```

Comment:
Do similar change in content/index.html to keep the pages in sync

Reply:
Done